tools/src/Transforms.ts

```diff
@@ -101,6 +101,26 @@ export async function transformFileAsync(
   await fs.outputFile(filePath, transformString(content, transforms));
 }
 
+/**
+ * Transforms multiple files' content in-place.
+ */
+export async function transformFilesAsync(files: string[], transforms: FileTransform[]) {
```

Comment:
It seems to be copied from `copyFileWithTransformsAsync` 😅 (except the `outputFile`). We should reuse some parts of that, especially filtering an array of transforms to apply.
Maybe make it for a single file and add optional parameter with a custom target path that defaults to the original file.

Reply:
from https://github.com/expo/expo/pull/19078/commits/21bc6d175837af8eaf3fdf301cf8840011ef1d60, i move the filtering into `getTransformedFileContentAsync`, because the `getTransformedFileContentAsync` is only be used in `transformFilesAsync` and `copyFileWithTransformsAsync`.

Reply:
Cool, now it is good enough I think 😉